dist/index.js

```diff
@@ -8,6 +8,11 @@ module.exports = scan
 module.exports.cli = scanCLI
 module.exports.customScanner = scanUsingCustomSonarQubeScanner
 
+let foo
+if (foo == '') {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace "==" with "===". ([javascript:EqEqEq](https://sonarcloud.io/organizations/default/rules#rule_key=javascript%3AEqEqEq))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=sonarqube-scanner&pullRequest=38&issues=AWLNyHlbjr7wspRoiHiN&open=AWLNyHlbjr7wspRoiHiN)

[ISSUE_KEY]: AWLNyHlbjr7wspRoiHiN